### ISSUE TITLE
Idea: Saving pfx to keyvault

### ISSUE THREAD

**Issue by egamitilps** [n3wt0n/AzureWebAppSSLManager#10]
This is exactly what i'm looking for.
An enhancement I'd love to see is saving the pfx to keyvault along with the password.

Awesome work!

**Comment by n3wt0n** [comment]
Good idea, I could add a setting for it so the user can decide if saving to keyvault or not.

Thanks :)

**Comment by hendriksteinhorst** [comment]
may I ask if there are any updates on this topic as I would love to see this feature too

**Comment by n3wt0n** [comment]
@hendriksteinhorst I still have plans to add that to the service, but I've been super busy and haven't had enough time to do this properly. As you can imagine it's a feature that needs to be done with the right amount of attention to be sure everything works properly.

I'm gonna do it as soon as I can.

**Comment by hendriksteinhorst** [comment]
I have done it manually recently to use the certificate in Azure Front Door. Here is what I learned:
- Azure Front Door accepts only certificates with RSA private key (see line 145 in CertificatesHelper.cs)
- the certificate password must be empty, otherwise Azure Front Door can't import it

I don't know the security implications of these changes as I'm no expert in certificates.
